### PR TITLE
refactor(tracker): update trackers behavior and introduce activities and audiences

### PR DESCRIPTION
**Pull request by skyborgff**

### CODE REVIEW

**Comment by skyborgff** [comment]
It should be roughly in line with this spec: https://github.com/animeshon/apis-tracker/issues/4
Comments / improvements related to the spec should be done there, as to  keep the comments in one place

File: animeshon/tracker/v1alpha1/tracker.proto [hunk 1]
```diff
@@ -332,4 +340,49 @@ message ExportTrackersRequest {
...
+  // of the operation. Operations that have successfully been cancelled
+  // have [Operation.error][] value with a [google.rpc.Status.code][google.rpc.Status.code] of 1,
+  // corresponding to `Code.CANCELLED`.
+  bool requested_cancellation = 6 [(google.api.field_behavior) = OUTPUT_ONLY];
+
+  // Output only. API version used to start the operation.
+  string api_version = 7 [(google.api.field_behavior) = OUTPUT_ONLY];
+
+  // Output only.
+  int32 step_count = 8 [(google.api.field_behavior) = OUTPUT_ONLY];
```
Comment (Asuka-Langley-Soryu) [inline animeshon/tracker/v1alpha1/tracker.proto:372]: from here on you don't need the fields, those were crossrefs-specific.

Comment (skyborgff) [reply animeshon/tracker/v1alpha1/tracker.proto:372]: but shouldn't we send a percentage, or a progress value?

Comment (Asuka-Langley-Soryu) [reply animeshon/tracker/v1alpha1/tracker.proto:372]: then you only need a single field `int32 progress_percentage = 8;`

**Review by christian-roggia — APPROVED** [review]

**Pull request merged**